source/FWU/MBFW/chapter2-uefi.md

```diff
@@ -35,41 +87,42 @@ The UEFI specification defines a mapping between the ESRT fields and the `EFI_FI
 `LastAttemptStatus` is expected to be maintained by the UEFI implementation, relying partially on information provided by early platform boot stages [FWU]_.
 
 The acceptance status of each FW image is provisionally displayed in the `LastAttemptedStatus` field [[1]](#provisional) in the ESRT image entry.  
-A value of `0x3fff` implies that the image has not been accepted. The OS must explicitly accept the image by installing an acceptance capsule as described in `OS directed FW image acceptance`_.
+A value of `0x3fff` implies that the image has not been accepted. The OS must explicitly accept the image by installing an acceptance capsule or set ABindications as described in `OS directed FW image acceptance`_.
 
 [provisional]: https://gitlab.com/Linaro/trustedsubstrate/mbfw/-/blob/master/source/chapter2-uefi.rst?ref_type=heads&plain=0#id7
 >[[1]](#provisional) Presenting the image acceptance status in the LastAttemptedStatus field is a provisional arrangement. A more permanet solution is under discussion.
```

Comment:
Should this comment be removed now? We discussed it and no other options were proposed.